### DISCUSSION TITLE
Any room for memory usage improvment?

### DISCUSSION THREAD

**Discussion by kawpii** [tauri-apps/tauri#3162] [Q&A] [answered]
According to [these benchmarks](https://github.com/Elanis/web-to-desktop-framework-comparison#benchmarks) Neutralinojs have way better memory usage? Anyone know why? Is there any room for improvement for Tauri?

**Comment by FabianLars** [comment, marked as answer]
Well, the memory benchmarks are actually just wrong or at least measured in an inconsistent way.
Kinda bad news, but even the tauri one is higher than that. for example my somewhat simple app uses ~120MB. But only ~5MB is the actual tauri/rust process, the rest is WebView2.
And that 5MB is the important bit here, because that's what was measured in the Neutralino benchmark (2MB).
Neutralino uses WebView2 just like we do, so they have to have similar memory usage, aside from the main process (2mb vs 5mb)🤷 

While we're at it, The electron app from that repo is around 85MB for me, but you definitely won't see that in the wild lol.

About memory improvements in general. Probably not, we have no real control about the webview memory usage, so that's something that has to be improved upstream (but probably won't be, because "memory is meant to be used")
And trying to improve that 5MB of "our" process doesn't sound worth the effort at all, especially since it's possible that there isn't room for improvement.

**Reply by kawpii** [reply]
> And that 5MB is the important bit here, because that's what was measured in the Neutralino benchmark (2MB).

That wrong indeed! I did not look at the source


> And trying to improve that 5MB of "our" process doesn't sound worth the effort at all,

3MB difference is neglectable.
